stargate/crates/stargate-lib/src/request_pipeline/executor.rs

```diff
@@ -47,7 +47,7 @@ fn execute_node<'schema, 'request>(
     context: &'request ExecutionContext<'schema, 'request>,
     node: &'request PlanNode,
     results: &'request RwLock<Value>,
-    path: &'request ResponsePath,
+    path: &'request [String],
```

Comment:
not clear why this type change? seems unrelated to removing unwraps

Reply:
Recommendation (that I agree with) from Clippy.
`ResponsePath` is a type alias for `Vec<String>`, using a slice `&[String]` is better than expecting a `&Vec<String>` -- it saves an allocation and allows the function to be called with an array.